lib/mini_scheduler.rb

```diff
@@ -52,24 +52,28 @@ def self.skip_schedule(&blk)
   end
 
   def self.start
-    manager = Manager.new
+    Manager.discover_queues.each do |queue|
+      manager = Manager.new(queue: queue)
 
-    Manager.discover_schedules.each do |schedule|
-      manager.ensure_schedule!(schedule)
-    end
+      Manager.discover_schedules.each do |schedule|
```

Comment:
Small perf remark: should we move that outside the loop and re-use it? Something like

``` ruby
schedules = Manager.discover_schedules

Manager.discover_queues.each do |queue|
  schedules.each do |schedule|
    manager.ensure_schedule!(schedule) if schedule.queue == queue
  end
end
```